.github/actions/build-external/action.yml

```diff
@@ -8,27 +8,41 @@ inputs:
   version:
     description: "Version of the dependency to build"
     required: true
+  host-build:
+    description: "Whether to build a host library for use in cross-compiling (e.g. a Linux build of Qt that gets used during the WASM build process)"
+    required: false
+    default: false
 
 outputs:
   path:
-    description: "Path to the installed dependency"
+    description: "Absolute path to the installed dependency"
     # FIXME: collapse this with cache-check.path
-    value: build/external/${{ inputs.library }}-${{ inputs.version }}
+    value: ${{ github.workspace }}/build/external/${{ inputs.host-build == 'true' && 'host/' || '' }}${{ inputs.library }}-${{ inputs.version }}
```

Comment:
I wonder if `build/external/${{ inputs.host-build == 'true' && 'host/' || '' }}` can get factored out in some way.

I don't know how to manipulate the action yml well enough to make a useful suggestion, so I'm fine with leaving this based on my own ignorance.

Reply:
Yeah, I'm not a huge fan of the way this is written either, since that same value is also set separately in both of the steps below.  I tried to figure out some way to clean this up or at least remove the duplication, but I couldn't get anything to work.  My initial thought was to create an `env` block that would apply to the whole file, but the contents of `env` blocks appear to only get evaluated once, not once per call into the file, so anything that needs to vary per-build winds up wrong.  The GitHub actions documentation wasn't any help here, so I eventually gave up.